client/my-sites/migrate/step-import-or-migrate.jsx

```diff
@@ -138,6 +138,7 @@ class StepImportOrMigrate extends Component {
 					<CardHeading>{ translate( 'What do you want to import?' ) }</CardHeading>
 
 					{ this.getJetpackOrUpgradeMessage() }
+					{ /* TODO: is the following code in use / up to date? */ }
```

Comment:
Is the following "import everything" behaviour up to date? I don't have enough context to understand if it can stay as-is, or if it can be deleted, or otherwise tweaked.